apps/storybook/src/comboboxes/Autocomplete.stories.tsx

```diff
@@ -237,17 +230,11 @@ export const Controlled: Story<(typeof books)[number]> = {
           <AutocompleteTrigger placeholder="Search a book" />
 
           <AutocompleteContent>
-            <AutocompleteList>
-              {items.map((item) => (
-                <AutocompleteItem
-                  addonAfter={<AutocompleteItemIndicator />}
-                  item={item}
-                  key={item.id}
-                >
-                  {item.title}
-                </AutocompleteItem>
-              ))}
-            </AutocompleteList>
+            {items.map((item) => (
+              <AutocompleteItem asChild item={item} key={item.id}>
+                <ListboxRadioItem>{item.title}</ListboxRadioItem>
+              </AutocompleteItem>
```

Comment:
Vai, did we use the `ListboxRadioItem` intentionally here? Can't we use the `AutocompleteRadioItem`?

Reply:
nope - replaced